java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -1847,9 +1534,7 @@ public ExpressionTree newAnnotatedParameterizedIdentifier(
 
     ExpressionTree result = new IdentifierTreeImpl((InternalSyntaxToken) identifierToken);
 
-    if (annotations.isPresent()) {
-      ((JavaTree) result).prependChildren(annotations.get());
-    }
+    // FIXME SONARJAVA-547 Handle dimensions
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)